js/components/user-bar.js

```diff
@@ -125,10 +126,10 @@ define([
 							if (job.status() !== n.status) {
 								job.status(n.status);
 								job.viewed(false);
-								job.duration = duration;
-								job.endDate = displayedEndDate;
+								job.duration(duration);
+								job.endDate(displayedEndDate);
 								job.url = jobDetailsService.getJobURL(n);
```

Comment:
Why is this variable remains not observable when all others were migrated to the observables?